queue_services/business-bn/src/business_bn/config.py

```diff
@@ -0,0 +1,138 @@
+    """
+
+    SERVICE_NAME = "business-bn"
+    PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
+
+    LD_SDK_KEY = os.getenv("LD_SDK_KEY", None)
+    COLIN_API = f"{os.getenv('COLIN_API_URL', '')}{os.getenv('COLIN_API_VERSION', '')}"
+
+    SEARCH_API = (
+        f"{os.getenv('REGISTRIES_SEARCH_API_INTERNAL_URL', '')}{os.getenv('REGISTRIES_SEARCH_API_VERSION', '/api/v1')}"
+    )
+
+    # service accounts
+    ACCOUNT_SVC_AUTH_URL = os.getenv("ACCOUNT_SVC_AUTH_URL")
+    ACCOUNT_SVC_CLIENT_ID = os.getenv("ACCOUNT_SVC_CLIENT_ID")
+    ACCOUNT_SVC_CLIENT_SECRET = os.getenv("ACCOUNT_SVC_CLIENT_SECRET")
+    ACCOUNT_SVC_TIMEOUT = os.getenv("ACCOUNT_SVC_TIMEOUT")
+
+    BN_HUB_API_URL = os.getenv("BN_HUB_API_URL", None)
+    BN_HUB_CLIENT_ID = os.getenv("BN_HUB_CLIENT_ID", None)
+    BN_HUB_CLIENT_SECRET = os.getenv("BN_HUB_CLIENT_SECRET", None)
+    BN_HUB_MAX_RETRY = int(os.getenv("BN_HUB_MAX_RETRY", "9"))
+    TEMPLATE_PATH = os.getenv("TEMPLATE_PATH", "src/business_bn/bn_templates")
+
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+    # POSTGRESQL
+    DB_USER = os.getenv("DATABASE_USERNAME", "")
+    DB_PASSWORD = os.getenv("DATABASE_PASSWORD", "")
+    DB_NAME = os.getenv("DATABASE_NAME", "")
+    DB_HOST = os.getenv("DATABASE_HOST", "")
+    DB_PORT = os.getenv("DATABASE_PORT", "5432")
+    SQLALCHEMY_DATABASE_URI = f"postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"
+
+    # legislative timezone for future effective dating
+    LEGISLATIVE_TIMEZONE = os.getenv("LEGISLATIVE_TIMEZONE", "America/Vancouver")
+
+    # Pub/Sub
+    GCP_AUTH_KEY = os.getenv("GCP_AUTH_KEY", None)
+    AUDIENCE = os.getenv("AUDIENCE", "https://pubsub.googleapis.com/google.pubsub.v1.Subscriber")
+    PUBLISHER_AUDIENCE = os.getenv("PUBLISHER_AUDIENCE", "https://pubsub.googleapis.com/google.pubsub.v1.Publisher")
+    SUB_AUDIENCE = os.getenv("SUB_AUDIENCE", "")
+    SUB_SERVICE_ACCOUNT = os.getenv("SUB_SERVICE_ACCOUNT", "")
+    SBC_CONNECT_GCP_QUEUE_DEBUG = os.getenv("SBC_CONNECT_GCP_QUEUE_DEBUG", "")
+
+    BUSINESS_EVENTS_TOPIC = os.getenv("BUSINESS_EVENTS_TOPIC", "business-bn")
+    BUSINESS_EMAILER_TOPIC = os.getenv("BUSINESS_EMAILER_TOPIC", "business-bn-emailer")
+
+
+class DevConfig(Config):  # pylint: disable=too-few-public-methods
+    """Creates the Development Config object."""
+
+    TESTING = False
+    DEBUG = True
+
+
+class TestConfig(Config):  # pylint: disable=too-few-public-methods
+    """In support of testing only.
+
+    Used by the py.test suite
+    """
+
+    DEBUG = True
+    TESTING = True
+    # POSTGRESQL
+    DB_USER = os.getenv("DATABASE_TEST_USERNAME", "")
+    DB_PASSWORD = os.getenv("DATABASE_TEST_PASSWORD", "")
+    DB_NAME = os.getenv("DATABASE_TEST_NAME", "")
+    DB_HOST = os.getenv("DATABASE_TEST_HOST", "")
+    DB_PORT = os.getenv("DATABASE_TEST_PORT", "5432")
+    SQLALCHEMY_DATABASE_URI = f"postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"
+
```

Comment:
I think GCP_AUTH_KEY should be set to None here too so that we're certain nothing gets sent out during a test run